Gifski/MainWindowController.swift

```diff
@@ -320,8 +337,38 @@ extension MainWindowController: NSMenuItemValidation {
 		switch menuItem.action {
 		case #selector(open)?:
 			return !isRunning
+		case #selector(quickLook(_:))?:
+			return conversionCompleted
 		default:
-			return validateMenuItem(menuItem)
+			return true
+		}
+	}
+}
+
```

Comment:
Can you also implement `QLPreviewPanelDelegate` so we get the "zoom" effect when pressing Space?

Reply:
Alright, will try! Is there somewhere in a code a thumbnail I could use or do I need to create it myself?

Reply:
The thumbnail shown in the conversion completed view. It should zoom animate from that one.

Reply:
Done 👍